.travis.yml

```diff
@@ -23,7 +23,9 @@ install: true
 script:
   - export GO15VENDOREXPERIMENT=1
   - if [ $TRAVIS_OS_NAME == 'linux' ]; then
-      cd shared && npm i --global flow-bin@`tail -n1 .flowconfig` && flow &&
+      cd shared && npm i -g flow-bin@`tail -n1 .flowconfig` && flow &&
+      cd ../desktop && npm i -g babel-cli && npm run setup-ci-tools && npm run setup-dev-tools &&
+      cd ../shared && eslint . && 
```

Comment:
Is this running eslint in the `shared` directory? I believe it should be run out of the top level directory, since that's where the eslint config resides.